index.html

```diff
@@ -6470,39 +6470,25 @@ <h3>Executing Script</h3>
 <dl class=switch>
  <dt><a>undefined</a>
  <dt><a><code>null</code></a>
- <dd><p><a>Success</a> with data <a><code>null</code></a>.
+ <dd><p><p>Return <a>success</a> with data <a><code>null</code></a>.
```

Comment:
If we're fixing inconsistencies anyway, we usually just write "`null`" rather than using `<code>` or linking it.

Reply:
If we make such a change we should do that for the full document at once. This drastically enlarges the scope of this PR and we should be better of doing it in a follow up PR if really needed.

Reply:
Ah yeah, this is apparently different compared to BiDi and other specs. We don't need to fix it here.